README.md

```diff
@@ -1 +1,144 @@
-# hellomicroverse
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [📝 License](#license)
+
+
+
+# 📖 [Hello Microverse] <a name="First microverse project"></a>
+
+> This one is My first Microverse project. In this project i am using html and css. I learned about linters and npm.
+
+**[Hello Microverse]** is a basic project.
+
+## 🛠 Built With <a name="HTML & CSS"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html5.org/">HTML</a></li>
+    <li><a href="https://css.org/">CSS</a></li>
+  </ul>
+</details>
+
+
+
+
+
+### Key Features <a name="key-features"></a>
+
+- **[text-color]**
+- **[text-styles]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+. A code editor of your choice(like vs code or Atom and so on)
+. Version control System (git is preferred)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+cd my-folder
+git clone https://github.com/tajulafreen/hellomicroverse.git
+cd hellomicroverse
```

Comment:
[OPTIONAL] Well done here 💯 but this can be formatted better, how about you put it in a format so that the user can copy it? 😉 You can do that by adding ``` on top of it and then the same thing under the text it then becomes like this 👇🏽 

```
cd my-folder
git clone https://github.com/tajulafreen/hellomicroverse.git
cd hellomicroverse
```